psutil/psutil.go

```diff
@@ -115,16 +126,34 @@ func (p *Psutil) GetMetricTypes(_ plugin.ConfigType) ([]plugin.MetricType, error
 		return nil, err
 	}
 	mts = append(mts, mts_...)
+	mts = append(mts, getDiskUsageMetricTypes()...)
 
 	return mts, nil
 }
 
 //GetConfigPolicy returns a ConfigPolicy
 func (p *Psutil) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
 	c := cpolicy.New()
+	config := cpolicy.NewPolicyNode()
+	r, err := cpolicy.NewStringRule("mount_points", false)
+
+	if err != nil {
+		return nil, err
+	}
+	r.Description = "Mountpoints"
+	config.Add(r)
+	c.Add([]string{""}, config)
```

Comment:
This line enforces user to provide configuration via global config, while it seems like it's needed only during collection.